daterangepicker.js

```diff
@@ -1308,8 +1315,10 @@
             }
 
             //if a new date range was selected, invoke the user callback function
-            if (!this.startDate.isSame(this.oldStartDate) || !this.endDate.isSame(this.oldEndDate))
+            if ((!this.startDate.isSame(this.oldStartDate) || !this.endDate.isSame(this.oldEndDate))
```

Comment:
The changes are laser-focused and looking good. Please increase the versions in package and bower JSON files